moc_openstack/manifests/cronjob.pp

```diff
@@ -0,0 +1,62 @@
+class moc_openstack::cronjob (
+  $base_dir  = '/etc/yum.repos.d/',
+  $repo_server = '127.0.0.1',
+) {
+
+  if $::environment == 'production' {
+    $rhel7_file_path = "${base_dir}rhel7local-prod.repo"
+    $epel_file_path = "${base_dir}epel7local-prod.repo"
+    $suricata_file_path = "${base_dir}suricata7local-prod.repo"
+    $rhel7_link = "http://${repo_server}/repos/rhel7local-prod.repo"
+    $epel_link = "http://${repo_server}/repos/epel7local-prod.repo"
+    $suricata_link = "http://${repo_server}/repos/suricata7local-prod.repo"
+  } else {
+    $rhel7_file_path = "${base_dir}rhel7local.repo"
+    $epel_file_path = "${base_dir}epel7local.repo"
+    $suricata_file_path = "${base_dir}suricaa7local.repo"
```

Comment:
"/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel*.repo" 
would disable the local epel repos as well and this is not what we want. That needs to become 2 lines disabling only the default epel repo files:
"/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel.repo" 
"/bin/sed -i '/enabled/c\enabled = 0 ' /etc/yum.repos.d/epel-testing.repo"